core/src/components/title/title.ios.scss

```diff
@@ -31,7 +31,7 @@
 
   position: relative;
 
-  font-size: 13px;
+  font-size: dynamic-font-max(13px, 1.8);
```

Comment:
I used the Contacts app as a reference.

Default
![Screenshot 2023-08-29 at 5 00 42 PM](https://github.com/ionic-team/ionic-framework/assets/13530427/e2027d57-8cc6-4e27-9525-616c08fd8423)

AX5
![Screenshot 2023-08-29 at 4 59 19 PM](https://github.com/ionic-team/ionic-framework/assets/13530427/ce225cdf-9106-4ad4-980e-4643006ae0f7)